package.json

```diff
@@ -27,6 +27,7 @@
     "core-js": "^3.2.1",
     "rxjs": "^6.5.2",
     "three": "^0.108.0",
+    "three.meshline": "^1.2.0",
```

Comment:
@ComFreek what do you think about adding [three.meshline](https://github.com/spite/THREE.MeshLine) dependency? Or it's better to keep "clean" repo without additional libraries?

Reply:
@makimenko I think users of atft don't ship mission-critical software anyway. I think #1 use case for atft: prototyping and playing around, perhaps creating some cool tech demo (do you agree?). Hence I suppose a thresh.meshline dependency would be OK.